aio/aio-proxy/aio_proxy/tests/e2e_tests/test_api.py

```diff
@@ -0,0 +1,22 @@
+# importing the requests library
+import requests
+
+# api-endpoint
+base_url = "http://localhost:4500/"
+
+
+def test_fetch_company():
+    path = "search?q=ganymede"
+    response = requests.get(url=base_url + path)
+    response_json = response.json()
+    total_results = response_json["total_results"]
+    siren = response_json["results"][0]["siren"]
+    assert response.status_code == 200
+    assert total_results > 10
+    assert siren
+
```

Comment:
We could add a test which correlate the siren found with the result that we want at rank n°1

Reply:
But which one would that be ?
There are so many possible answers for "ganymede" :/
Unless I change request to "la poste" for example ?

Reply:
@geoffreyaldebert j'ai testé avec la poste.

Reply:
great